draft-ietf-quic-manageability.md

```diff
@@ -911,9 +911,24 @@ This document has no actions for IANA.
 
 # Security Considerations
 
-Supporting manageability of QUIC traffic inherently involves tradeoffs with the
-confidentiality of QUIC's control information; this entire document is therefore
-security-relevant.
+QUIC is an encrypted and authenticated transport. That means, once the
+cryptographic handshake is complete, QUIC endpoints discard most packets that
+are not authenticated, greatly limiting the ability of an attacker to interfere
+with existing connections.
```

Comment:
Perhaps note that on-path modifications of packets will result in one of two outcomes:

- packet ignored (and frames eventually retransmitted, if necessary)
- connection dropped